device/alpaka/include/traccc/alpaka/seeding/track_params_estimation.hpp

```diff
@@ -66,7 +66,7 @@ struct track_params_estimation
     /// Memory resource used by the algorithm
     traccc::memory_resource m_mr;
     /// Copy object used by the algorithm
-    ::vecmem::copy& m_copy;
+    vecmem::copy& m_copy;
```

Comment:
This was done by mistake, right? The current version (with `::vecmem`) should be the more robust setup.

Reply:
Slightly over-zealous refactoring when I moved from `traccc::alpaka::vecmem` to `traccc::alpaka::vecmem_resources`.

I'll revert.